api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 233**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events):
+  TODO: Event type renamed and hotspots array renamed.
+
```

Comment:
This should go on feature level 232 with the other changes. And please add the API documentation at this point; these API changes look good.